src/modules/handleTodo.js

```diff
@@ -0,0 +1,41 @@
+import setLocalStorage from './setLocalStorage.js';
+
+const todoContainer = document.querySelector('.todo-container');
+
+const handleTodo = (todo, todoItems) => {
+  const items = document.querySelectorAll('.todo');
+  items.forEach((item) => {
+    const idInNum = Number(item.querySelector('.left').getAttribute('data-time'));
+    if (idInNum === todo.ID) {
+      // // mark complete
+      // item.querySelector('.checkbox').addEventListener('change', () => {
```

Comment:
It would be nice if you deleted this unused code fr it makes your codebase look untidy. Lets stick and adhere to the micro verse clean code policy.